pkg/api/types.go

```diff
@@ -587,6 +587,8 @@ type PodSpec struct {
 	// the the scheduler simply schedules this pod onto that host, assuming that it fits
 	// resource requirements.
 	Host string `json:"host,omitempty"`
+	// Optional: Default to false.
```

Comment:
Comment on what this does.  Comment that anyone who sets this really should make sure to declare their ports.